.changeset/lazy-beds-matter.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-frontend/mc-scripts': patch
+---
+
+> These changes are only relevant when using the experimental Vite bundler (`ENABLE_EXPERIMENTAL_VITE_BUNDLER=true`).
+
+Allow to load a Vite config in addition to the default config. This is useful for using extra Rollup/Vite plugins, or other Vite options.
```

Comment:
So now if users want to use a custom config file they just need to create a `vite.config.js` in the root directory of their application?

Reply:
Yes that's the idea.